packages/typespec-autorest/src/lib.ts

```diff
@@ -193,6 +199,14 @@ const EmitterOptionsSchema: JSONSchemaType<AutorestEmitterOptions> = {
       description:
         "Determine whether and how to emit x-ms-long-running-operation-options for lro resolution",
     },
+    "emit-common-types-schema": {
+      type: "string",
+      enum: ["reference-only", "for-visibility-changes"],
```

Comment:
```suggestion
      enum: ["never", "for-visibility-changes"],
```

I think that makes more sense with the option name, alternatively could call teh option: `common-types-schema`

Reply:
I was thinking that we could use 'all' as a setting for actually emitting the common types schema in the future, but 'never' makes sense as well.